src/pages/index.js

```diff
@@ -23,6 +23,14 @@ import CloudNativeLeaders from "../sections/Home/Cloud-Native";
 import SoSpecial from "../sections/Home/So-Special-Section";
 
 const IndexPage = () => {
+  const [mounted, setMounted] = React.useState(false);
+  React.useEffect(() => {
+    setMounted(true);
+  }, []);
+  if (!mounted) {
+    return <div style={{ visibility: "hidden" }}>Prevent Flash</div>;
```

Comment:
Why is this required?

Reply:
I was trying to prevent this flashing effect on refresh. Saw this approach on #2910 

https://user-images.githubusercontent.com/85789734/179425531-9645e9ba-19dc-43e0-a952-81538379a6ec.mov

Reply:
Seems a little hacky 🤔

Reply:
That blip/flash behavior is even worse, though.